src/controllers/business/index.js

```diff
@@ -37,6 +37,8 @@ export const createBusiness = (obj, { input }, { models }) => {
       address: addressInput
     } = input
 
+    if (isEmpty(currentUser)) throw new Error('An existing user is required to create a business.')
+
```

Comment:
No need for this check. This resolver should be protected with an `isAuthenticated` check.